docs/DeveloperGuide.md

```diff
@@ -724,31 +722,31 @@ testers are expected to do more *exploratory* testing.
    1. Prerequisites: List all properties using the `list` command. Multiple properties in the list.
 
    2. Test case: `delete property 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First property is deleted from the list. Details of the deleted property shown.
 
    3. Test case: `delete property 0`<br>
-      Expected: No property is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No property is deleted. Error message shown.
 
-   4. Other incorrect delete commands to try: `delete`, `delete property x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+   4. Other incorrect delete commands to try: `delete`, `delete property x` (where x is larger than the list size)<br>
+      Expected: Similar to previous test cases.
 
 4. Sort properties that are in the displayed list
    1. Test case: `sort property name desc`<br>
       Expected: Sorted all properties in the displayed properties list by name in descending order.
    2. Test case: `sort property name`<br>
-      Expected: There is no sort direction given. Properties are not sorted.
+      Expected: As there is no sort direction given, properties are not sorted.
 
 ### Exporting and importing data
 
-1. Testing properties export/import csv
+1. Testing properties export/import CSV
    1. Remove `propertywhiz.json` file in the /data folder.
    2. Run `PropertyWhiz` to obtain an initialized list of properties and buyers.
-   3. Enter Command: `export properties`, select a csv file at a convenient location..<br>
-      Expected: csv file is created, containing a header and 1 property per row.
+   3. Enter Command: `export properties`, select a CSV file at a convenient location.<br>
+      Expected: CSV file is created, containing a header and 1 property per row.
    4. Enter Command: `clear` to clear existing properties and buyers.
-   5. Enter Command: `import properties` from the csv file.<br>
+   5. Enter Command: `import properties` from the CSV file.<br>
       Expected: Properties are successfully imported.
-   6. Enter Command: `import properties` from the csv file.<br>
+   6. Enter Command: `import properties` from the CSV file.<br>
       Expected: Import properties failed due to duplicates.
 2. Repeat the commands above to test for buyers.
```

Comment:
For the section **Testing multiple commands quickly**, add one more `back` command at line 808